blacklist-call/functions/blacklist-call.js

```diff
@@ -0,0 +1,19 @@
+exports.handler = function(context, event, callback) {
+    // List all blocked phone numbers in quotes and E.164 formatting, separated by a comma
+    let blacklist = event.blacklist;  
+    let twiml = new Twilio.twiml.VoiceResponse();
+    let blocked = true;
+    if (blacklist.length > 0) {
```

Comment:
Right now this is going to be a string. It works but it's not great. Could we update this to transform into an array first:

```
const blacklistArray = blacklist.split(',').map(num => number.trim())
```

And then update the rest of the code to use `blacklistArray`. Line 7 can then be `blacklistArray.includes(event.From)`